package.json

```diff
@@ -6,12 +6,14 @@
   "scripts": {
     "start": "node src/server/server.js",
     "test": "jest",
-    "seed": "time node ./data/makeDataScript.js",
+    "makeAndSeed": "time node ./data/makeData.js",
```

Comment:
just a suggestion, you could combine the code into one file, or add a && so you can use just one script instead of 2 or 3.